pkg/tsdb/prometheus/querydata/response.go

```diff
@@ -112,7 +115,6 @@ func addMetadataToMultiFrame(q *models.Query, frame *data.Frame, enableDataplane
 	if frame.Meta == nil {
 		frame.Meta = &data.FrameMeta{}
 	}
-	frame.Meta.ExecutedQueryString = executedQueryString(q)
```

Comment:
You need to delete this from `addMetadataToWideFrame` function too

Reply:
https://github.com/grafana/grafana/issues/73697

Reply:
Also "Wide" is just one frame, so it shouldn't matter.

Reply:
Removing as in removing the feature flag, and the behavior that the feature flag enables when turned on.